website/docs/r/instance_ip.html.markdown

```diff
@@ -20,7 +20,7 @@ resource "scaleway_instance_ip" "server_ip" {}
 The following arguments are supported:
 
 - `reverse` - (Deprecated) Please use the `scaleway_instance_ip_reverse_dns` resource instead.
-- `server_id` - (Optional) The ID of the server you want to attach this resource to.
+- `server_id` - (Deprecated) The ID of the server you want to attach this resource to.
```

Comment:
Add `Deprecated in favor of scaleway_instance_server.ip_id`
If possible add a lik to the correct page on ` scaleway_instance_server.ip_id`

Reply:
:+1: